src/teal-monaco-language.ts

```diff
@@ -0,0 +1,134 @@
+/* eslint-disable */
+// Based on https://raw.githubusercontent.com/microsoft/monaco-languages/master/src/lua/lua.ts
+
+import monaco from "monaco-editor";
+
+import IRichLanguageConfiguration = monaco.languages.LanguageConfiguration;
+import ILanguage = monaco.languages.IMonarchLanguage;
+
+export const tealMonacoLanguageConfiguration: IRichLanguageConfiguration = {
+  comments: {
+    lineComment: '--',
+    blockComment: ['--[[', ']]'],
```

Comment:
this is cool, is this duplicated from the vscode extension? just curious in case we need to make sure to sync them from time to time or create an npm package in the future.

Reply:
No, the vscode extension uses a TextMate grammar instead. I duplicated https://github.com/microsoft/monaco-languages/blob/master/src/lua/lua.ts.